source/config/configSpec.py

```diff
@@ -54,6 +54,9 @@
 	WASAPI = featureFlag(optionsEnum="BoolFlag", behaviorOfDefault="enabled")
 	soundVolumeFollowsVoice = boolean(default=false)
 	soundVolume = integer(default=100, min=0, max=100)
+	soundSplitState = integer(default=0)
+	# This is JSON-encoded list of integers.

```

Comment:
Can't this use `int_list` from configobj?

Reply:
Agree with @lukaszgo1's suggestion. Unless there is a good reason to do otherwise, using `int_list` would be consistent with what is done for speech mode, thus easier to understand quickly.

Reply:
> Can't this use `int_list` from configobj?

I haven't found a way to assign a non-empty default value to intList - do you know how to do so?

Reply:
This can be written as follows:
`includedSoundSplitModes =int_list(default=list(0, 1, 3))`

Reply:
Thanks, that works!